### DISCUSSION TITLE
how to setup OpenSUSE auto sign?

### DISCUSSION THREAD

**Discussion by Seele0oO** [Foxboron/sbctl#372] [General]
after running 
```
sudo zypper update
```
```
sudo sbctl verify
```
the output is
```
✗ /boot/efi/EFI/opensuse/grubx64.efi is not signed
```

how can I auto resign it after `zypper` `in` or `update`

**Comment by Foxboron** [comment]
Moved to discussions. This isn't a `sbctl` issue but a packaging "issue" with the OpenSUSE package. I don't know the answer to this.

**Comment by Seele0oO** [comment]
I solve it.

https://blog.20220228.xyz/blog/2024/09/02/opensuse-tumbleweed-enable-secure-boot-with-sbctl-and-dracut/
